### ISSUE TITLE
Integrated Dynamics GUI Variable Slot Interactable Area Offset from Visual

### ISSUE THREAD

**Issue by no-name-provided** [FTBTeam/FTB-Modpack-Issues#6474]
### Modpack

FTB Evolution

### Modpack version

1.5.0

### Launcher

FTB App

### Has the pack been modified

Yes

### Describe the bug

At first, it seemed like ID wasn't useable. Then, I spent a few minutes spam clicking and figured out that there was an offset issue. Feel free to pass along to the ID devs if this is on their end.

### Steps to reproduce

1. Place cable
2. Place WBI on cable
3. Program block card
4. Place block card in "Pick up block" slot in WBI
5. Realize it's still in your hand

### Expected behaviour

1. Place cable 2. Place WBI on cable 3. Program block card 4. Place block card in "Pick up block" slot in WBI 5. Be happy you suceeded

### Screenshots

Clicking here does nothing
![image](https://github.com/user-attachments/assets/818e121f-9f8b-43ee-8187-fd63400e83c1)

Clicking here-ish places the variable (it's finnicky)
![image](https://github.com/user-attachments/assets/58335935-44dc-4890-b3c4-c7ab0d9fb328)

### Additional information

Running with https://www.curseforge.com/minecraft/mc-mods/alltheleaks

**Comment by no-name-provided** [comment]
Can confirm that AllTheLeaks is not involved here.

**Comment by Gaz492** [comment]
Note for pack devs: It looks like something is interfering with the mod as this doesn't happen when testing with only integrated tunnels.
[](!skip)

**Comment by ArtpokeBr** [comment]
Yeah, this is still a thing, perhaps is something similar to what's happening in OB2? If I recall correctly placing variables there is also a bit finickiy...

Tested and can confirm that although it works, it requires multiple clicks and its not exactly responsive.

**Comment by no-name-provided** [comment]
Following a series of unfortunate interactions with Saereth, and pursuant to his request to "fuck right off", I am unsubscribing from this issue.
